integration-test/src/test/java/org/sagebionetworks/IT610BridgeData.java

```
@@ -0,0 +1,263 @@
+	private static BridgeClient bridge = null;
+	private static BridgeClient bridgeTwo = null;
+	private static List<Long> usersToDelete;
+
+	public static final int PREVIEW_TIMOUT = 10 * 1000;
+
+	public static final int RDS_WORKER_TIMEOUT = 1000 * 60; // One min
+
+	private List<String> handlesToDelete;
+
+	private static BridgeClient createBridgeClient(SynapseAdminClient client) throws Exception {
+		SynapseClient synapse = new SynapseClientImpl();
+		usersToDelete.add(SynapseClientHelper.createUser(adminSynapse, synapse));
+
+		BridgeClient bridge = new BridgeClientImpl(synapse);
+		bridge.setBridgeEndpoint(StackConfiguration.getBridgeServiceEndpoint());
+
+		// Return a proxy
+		return BridgeProfileProxy.createProfileProxy(bridge);
+	}
+
+	private static SynapseClient createSynapse(BridgeClient bridge) {
+		SynapseClient synapse = new SynapseClientImpl(bridge);
+		SynapseClientHelper.setEndpoints(synapse);
+		return synapse;
+	}
+
+	@BeforeClass
+	public static void beforeClass() throws Exception {
+		usersToDelete = new ArrayList<Long>();
+		
+		adminSynapse = new SynapseAdminClientImpl();
+		SynapseClientHelper.setEndpoints(adminSynapse);
+		adminSynapse.setUserName(StackConfiguration.getMigrationAdminUsername());
+		adminSynapse.setApiKey(StackConfiguration.getMigrationAdminAPIKey());
+		
+		bridge = createBridgeClient(adminSynapse);
+		bridgeTwo = createBridgeClient(adminSynapse);
+	}
+
+	@Before
+	public void before() throws SynapseException {
+		handlesToDelete = new ArrayList<String>();
+		
+		for (String id : handlesToDelete) {
+			try {
+				adminSynapse.deleteFileHandle(id);
+			} catch (SynapseNotFoundException e) { }
+		}
+	}
+	@After
+	public void after() throws Exception {
+	}
+
+	
+	@AfterClass
+	public static void afterClass() throws Exception {
+		for (Long id : usersToDelete) {
+			//TODO This delete should not need to be surrounded by a try-catch
+			// This means proper cleanup was not done by the test 
+			try {
```

please don't swallow exceptions